### PR TITLE
Quickstart tutorial: Add an integration test for the expected nodes

### PR DESCRIPTION
**Pull request by AndyZe**
### Description

This adds an integration test for the first tutorial presented to new users: [MoveIt Quickstart in RViz](https://moveit.picknik.ai/main/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html). It checks that the expected nodes and action servers are available.

I think this type of test is great to have, especially on the entry point to MoveIt for most new users. There are quite a few issues this type of test would have helped to catch in the past including [[1](https://github.com/ros-planning/moveit2_tutorials/issues/385), [2](https://github.com/ros-planning/moveit2_tutorials/issues/308), [3](https://github.com/ros-planning/moveit2_tutorials/issues/193), [4](https://github.com/ros-planning/moveit_resources/pull/129)].

### Verification

You can verify the test passes locally like so:

`colcon test --packages-select moveit2_tutorials`

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,55 +5,36 @@ project(moveit2_tutorials)
...
+  rclcpp_action
+  rviz_visual_tools
+  tf2_geometry_msgs
+  tf2_geometry_msgs
+  tf2_ros
 )
 
+foreach(Dependency IN ITEMS ${THIS_PACKAGE_INCLUDE_DEPENDS})
+  find_package(${Dependency} REQUIRED)
+endforeach()
```
Comment (davetcoleman) [inline CMakeLists.txt:36]: That's a cool shortcut!

File: doc/tutorials/quickstart_in_rviz/CMakeLists.txt [hunk 2]
```diff
@@ -1 +1,24 @@
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(ros_testing REQUIRED)
+  find_package(Boost REQUIRED COMPONENTS filesystem)
+
+  # These don't pass yet, disable them for now
+  set(ament_cmake_copyright_FOUND TRUE)
```
Comment (davetcoleman) [inline doc/tutorials/quickstart_in_rviz/CMakeLists.txt:10]: Why doesn't the copyright test pass? Shouldn't that be easy to fix? Add a license file?

File: doc/tutorials/quickstart_in_rviz/test/bringup_test.cpp [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    }
+  }
+
+protected:
+  rclcpp::Node::SharedPtr node_;
+  rclcpp::Executor::SharedPtr executor_;
+  std::thread executor_thread_;
+};
+
+TEST_F(BringupTestFixture, BasicBringupTest)
```
Comment (davetcoleman) [inline doc/tutorials/quickstart_in_rviz/test/bringup_test.cpp:59]: Are there any other tests we could add here beyond bringup? Doesn't seem like it provides a ton of coverage

**Review by davetcoleman — COMMENTED** [review]
Looks pretty good to me, but I think someone who actually knows ROS2 tests should approve and merge.

File: doc/tutorials/quickstart_in_rviz/CMakeLists.txt (hunk 2 above)
Comment (AndyZe) [reply doc/tutorials/quickstart_in_rviz/CMakeLists.txt]: I just copy/pasted this from somewhere else, but actually all of these tests pass. :+1: Removed it.

**Comment by davetcoleman** [comment]
@vatanaksoytezer this looks good to me, is it ok if i merge it or do you want to take a look?

**Comment by AndyZe** [comment]
Give me a few minutes to add a trajectory test, please

**Comment by vatanaksoytezer** [comment]
> @vatanaksoytezer this looks good to me, is it ok if i merge it or do you want to take a look?

I just skimmed it real quick, generally looks good to me, feel free to merge

**Comment by AndyZe** [comment]
@vatanaksoytezer or @davetcoleman can I get an approval, then? I'll run CI one more time to be on the safe side

**Review by davetcoleman — APPROVED** [review]

**Pull request merged**